de/manual/ch_GUIMenus.xml

```diff
@@ -6658,535 +6658,1179 @@ url="&url-docs-de;guide/chapter_txns.html#txns-regstyle1"> &app; Kurs und Konzep
+                    </para>
+                  </entry>
 
-          <tbody>
-            <row>
-              <entry>
-                <para>&gmi.ac.re-pg;
-                </para>
-              </entry>
+                  <entry align="center" namest="c2" nameend="c3">
+                    <para>Beschreibung
+                    </para>
+                  </entry>
+                </row>
+              </thead>
+
+              <tbody>
+                <row>
+                  <entry namest="c1">
+                    <para>&gmi.vw.use-lay-cus;
+                    </para>
+                  </entry>
 
-              <entry>
-                <para>Die Bezeichnung der Registerkarte ändern.
-                </para>
-              </entry>
-            </row>
+                  <entry namest="c2" nameend="c3">
+                    <para>Verwende das aktuelle Layout für die Rechnungsansicht. Hiermit wird die Spaltenbreite der einzelnen
+                      Rechnungseinträge gespeichert.
+                    </para>
+                  </entry>
+                </row>
 
-            <row>
-              <entry>
-                <para>&gmi.ed.ven.edit;
-                </para>
-              </entry>
+                <row>
+                  <entry namest="c1">
+                    <para>&gmi.vw.res-lay-cus;
+                    </para>
+                  </entry>
 
-              <entry>
-                <para>Die Angaben des ausgewählten Lieferanten bearbeiten.
-                </para>
-              </entry>
-            </row>
+                  <entry namest="c2" nameend="c3">
+                    <para>Setzt das geänderte Layout auf die Voreinstellung zurück und aktualisiert die Anzeige.
+                    </para>
+                  </entry>
+                </row>
 
-            <row>
-              <entry>
-                <para>&gmi.bus.ven.new-bil;
-                </para>
-              </entry>
+                <row>
+                  <entry namest="c1">
+                    <para>&gsm.vw.ord;
+                    </para>
+                  </entry>
+
+                  <entry namest="c2" nameend="c3">
+                    <para>Legt die Sortierreihenfolge für die einzelnen Belegpositionen fest und zwar aufsteigend nach:
+                    </para>
+                  </entry>
+                </row>
+
+                <row>
+                  <entry namest="c2">
+                    <para>&gmi.vw.ord.def;
+                    </para>
+                  </entry>
+
+                  <entry namest="c3">
+                    <para>Gewöhnliche Rechnungsreihenfolge verwenden.
+                    </para>
+                  </entry>
+                </row>
+
+                <row>
+                  <entry namest="c2">
+                    <para>&gmi.vw.ord.date;
+                    </para>
+                  </entry>
+
+                  <entry namest="c3">
+                    <para>Sortierung nach Datum.
```

Comment:
Kein vollständiger Satz, also kein Punkt.
In Tabellen wäre ich für weglassen, in Listen für Semikoli.